feat(domains-dns): add troubleshooting index and doc (#4629)
scaleway/docs-content: pages/domains-and-dns/troubleshooting/cannot-transfer-domain.mdx

---
meta:
  title: I cannot transfer my domain
  description: This page helps you troubleshoot problems when you cannot transfer your domain
content:
  h1: I cannot transfer my domain
  paragraph: This page helps you troubleshoot problems when you cannot transfer your domain

> tags: failed-transfer cannot-transfer transfer-not-working transfer-unavailable

Reviewer: ```suggestion
tags: transfer domain troubleshooting error
```

dates:
  validation: 2025-03-14
  posted: 2025-03-14
categories:
  - network
---

This page shows you how to solve domain transfer issues related to **Scaleway Domains and DNS**.

<Macro id="requirements" />

  - A Scaleway account logged into the [console](https://console.scaleway.com)
  - [Created](/iam/how-to/create-api-keys/) an API key with sufficient [IAM permissions](/iam/reference-content/permission-sets/) to perform the actions described on this page
  - Registered a domain with Scaleway Domains and DNS

## Problem

You cannot transfer a domain.

## Possible causes

> - The top level domain (TLDs) you want to transfer are not supported by Scaleway. For example, Scaleway does not support the `.lat` or `.be` TLDs.

Reviewer: ```suggestion
- The top level domains (TLDs) you want to transfer are not supported by Scaleway. For example, Scaleway does not support the `.lat` or `.be` TLDs.
```

> - Your domain may be expiring soon. Some TLDs pevent a domain from being transferred before it expires.

Reviewer: ```suggestion
- Your domain may be expiring soon. Some TLDs prevent a domain from being transferred before it expires.
```

- You may have forgotten to initiate the transfer.

- Your domain may be locked at your former registrar.

- Your transfer code may be incorrect.

> - Your domain is premium, meaning it is short, usually associated with an actual industry or service, and often use a TLD such as `.com`, `.net`, or `.org`.

Reviewer: ```suggestion
- Your domain is premium, meaning it is short, usually associated with an actual industry or service, and often uses a TLD such as `.com`, `.net`, or `.org`.
```

- The DNS servers associated with the domain you want to transfer may still be pointing to those of the former provider.

## Solutions

> - Refer to [the list of TLDs supported by Scaleway](https://www.scaleway.com/en/domains-name/) or manually check that the TLD exists by following the 4 first steps documented in the [How to register an internal domain documentation](/domains-and-dns/how-to/register-internal-domain/). If the TLD you want to transfer does not display in the list of available domains, Scaleway does not support it.

Reviewer: ```suggestion
- Refer to [the list of TLDs supported by Scaleway](https://www.scaleway.com/en/domains-name/) or manually check that the TLD exists by following the 4 first steps documented in the [How to register an internal domain documentation](/domains-and-dns/how-to/register-internal-domain/). If the TLD you want to transfer is not displayed in the list of available domains, Scaleway does not support it.
```

- Renew your domain and try transferring it again.

- Make sure that you have initiated the transfer at your current registrar.

- Perform a [WHOIS lookup](https://who.is/) to find out what your domain's [EPP status code](https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en) is. If you see any of the following statuses in the WHOIS output, contact your registrar for assistance.

    - `clientTransferProhibited`: the current registrar has blocked the transfer. You must enable transfers in your domain settings.
    - `serverTransferProhibited`: the registry is preventing the transfer. Contact your registrar for details.

>     - `redemptionPeriod`: the domain has expired and is in a grace period before deletion. During this phase, transfers are not allowed. Howwever, the current owner can restore the domain by paying a redemption fee.

Reviewer: ```suggestion
    - `redemptionPeriod`: the domain has expired and is in a grace period before deletion. During this phase, transfers are not allowed. However, the current owner can restore the domain by paying a redemption fee.
```

- Scaleway does not support premium domains.

- Update the DNS servers to those of Scaleway (`ns0.dom.scw.cloud` and `ns1.dom.scw.cloud`). Once the DNS servers are updated, it may take up to 24 to 48 hours for the changes to propagate across the entire network.

## Additional information

When a domain is transferred to a new registrar, the DNS zone hosted by the previous registrar may be deleted almost immediately. This can cause an interruption in service if DNS records are not set up beforehand.

To avoid downtime, we recommend the following steps before initiating the transfer:

1. [Register your domain as external](/domains-and-dns/how-to/transfer-external-domain/) using Scaleway Domains and DNS. This will allow you to duplicate your existing DNS zone while the domain is still managed by the current registrar.
2. Make sure that the duplicated zone is correctly configured and matches the existing records at the current registrar.
3. Initiate the domain transfer to Scaleway Domaisn and DNS. Once the transfer is complete, your domain will automatically become an internal domain in Scaleway’s system, ensuring continuity of service.

> 

Reviewer: ```suggestion
```
